### PR TITLE
feat(APIM): mdm apim migration

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
APIM MDM endpoints migration

## Resolution
* Updated `.env.sample`
* Updated endpoints
* Updated GHA
* Update Docker file

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
 BASIC_AUTH_KEY=
 BASIC_AUTH_SECRET=
-MULESOFT_API_CIS_URL=
-MULESOFT_API_CIS_KEY=
-MULESOFT_API_CIS_SECRET=
-MULESOFT_API_MDM_EA_URL=
-MULESOFT_API_MDM_EA_KEY=
-MULESOFT_API_MDM_EA_SECRET=
+
+# APIM
```
Comment (ttbarnes) [inline .env.sample:25]: What does the "M" mean? Might be worth having full name here or somewhere?

File: .env.sample [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+MYSQL_ROOT_PASSWORD=
+DATABASE_URL=
+
+# API
+API_PORT=
+API_URL=
+SESSION_SECRET
+JWT_SIGNING_KEY=
+JWT_VALIDATING_KEY=
+MOCK_ACCOUNT_PASSWORD
```
Comment (ttbarnes) [inline .env.sample:16]: `MOCK_ACCOUNT_PASSWORD` is not specific to the API it's used in E2E and API

File: src/ui/.eslintrc.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ module.exports = {
     },
   },
   rules: {
-    'import/no-unresolved': [2, { caseSensitive: false }],
+    'import/no-unresolved': [1, { caseSensitive: false }],
```
Comment (ttbarnes) [inline src/ui/.eslintrc.js:33]: What's the reason for this change?

**Review by ttbarnes — COMMENTED** [review]
@abhi-markan Can you try rebuilding/compiling the UI please, we updated GOV library not long ago and the client side JS/CSS is minified, but seems to be reversed here 😢

File: .env.sample (hunk 1 above)
Comment (abhi-markan) [reply .env.sample:25]: APIM = API Management (This will be the product name)

File: src/ui/server/api/external/index.ts [hunk 4]
```diff
@@ -33,14 +31,8 @@ const external = {
...
-        url: `${process.env.MULESOFT_API_MDM_EA_URL}${MULESOFT_MDM_EA.CURRENCY}`,
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        auth: {
-          username: process.env.MULESOFT_API_MDM_EA_KEY,
-          password: process.env.MULESOFT_API_MDM_EA_SECRET,
-        } as AxiosBasicCredentials,
+        url: `${process.env.APIM_MDM_URL}${MULESOFT_MDM_EA.CURRENCY}`,
+        headers,
```
Comment (ttbarnes) [inline src/ui/server/api/external/index.ts:35]: Beautiful much simpler headers 😍

File: src/ui/server/constants/api.ts [hunk 5]
```diff
@@ -31,7 +31,8 @@ export const API = {
 
 export const EXTERNAL_API_ENDPOINTS = {
   MULESOFT_MDM_EA: {
-    CURRENCY: '/currency',
-    INDUSTRY_SECTORS: '/map-industry-sector?size=1000',
+    CURRENCY: '/currencies',
+    INDUSTRY_SECTORS: '/sector-industries',
+    MARKETS: '/markets?active=Y',
```
Comment (ttbarnes) [inline src/ui/server/constants/api.ts:36]: Active flag might not be required here, in the eligibility areas we want to display inactive/unsupported countries so we can render an appropriate message if a user selects an inactive/unsupportred country

File: .env.sample (hunk 2 above)
Comment (abhi-markan) [reply .env.sample:16]: Have corrected

File: src/ui/.eslintrc.js (hunk 3 above)
Comment (abhi-markan) [reply src/ui/.eslintrc.js:33]: Apologies have reverted back.

File: src/ui/server/constants/api.ts (hunk 5 above)
Comment (abhi-markan) [reply src/ui/server/constants/api.ts:36]: Have removed query parameter `active` with value `Y`

**Review by Zainzzkk — APPROVED** [review]

**Review by ttbarnes — APPROVED** [review]
